tools/backups2datalad/datasetter.py

```diff
@@ -81,11 +81,16 @@ def update_from_backup(
                 log.info("Pushing to sibling")
                 ds.push(to="github", jobs=self.config.jobs)
             if self.config.gh_org is not None:
-                stats = self.get_dandiset_stats(ds)
+                stats, substats = self.get_dandiset_stats(ds)
```

Comment:
Let's give it a more telling name 
```suggestion
                stats, zarrstats = self.get_dandiset_stats(ds)
```